bot.py

```diff
@@ -338,6 +338,147 @@ async def display_current_list_of_summoners(ctx):
         await ctx.send(embed=create_embed(embed_data))
 
 
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
```

Comment:
what are these that you are silencing?
`disable=too-many-locals, too-many-branches, too-many-statements`

Reply:
Silencing not useful pylint error messages that blocks push